be/src/exprs/agg/java_udaf_function.h

```diff
@@ -56,9 +54,13 @@ class JavaUDAFAggregateFunction : public AggregateFunction {
     void merge(FunctionContext* ctx, const Column* column, AggDataPtr __restrict state,
                size_t row_num) const override final {
         // TODO merge
-        DCHECK(!column->is_nullable());
-        DCHECK(column->is_binary());
-        const auto* input_column = down_cast<const BinaryColumn*>(column);
+        const BinaryColumn* input_column = nullptr;
+        if (column->is_nullable()) {
```

Comment:
should const null column and const column be taken into consideration?

Reply:
in aggregate_function input/output shouldn't be const column